Detectors/MUON/MID/Simulation/CMakeLists.txt

```diff
@@ -3,19 +3,21 @@ set(MODULE_NAME "MIDSimulation")
 O2_SETUP(NAME ${MODULE_NAME})
 
 set(SRCS
-   src/Detector.cxx
-   src/Materials.cxx
-   src/Materials.h
-   src/Geometry.cxx
-   src/Geometry.h
+  src/Detector.cxx
+  src/Geometry.cxx
+  src/Hit.cxx
+  src/Materials.cxx
+  src/Stepper.cxx
 )
 
 set(HEADERS
-   include/${MODULE_NAME}/Detector.h
+  include/${MODULE_NAME}/Detector.h
+  include/${MODULE_NAME}/Hit.h
+  include/${MODULE_NAME}/Stepper.h
```

Comment:
Does the Stepper.h include really need to be installed ?

Reply:
Hi @aphecetche ,
since Detector.h is installed, and the Stepper.h is a data member, I needed to install it as well.
However, this is an implementation detail, so I can avoid installing it if I properly set the include path in O2Dependencies.
This is done in the new commit.
Cheers,
Diego